### ISSUE TITLE
Which ConvNext did you use?

### ISSUE THREAD

**Issue by Richar-Du** [FudanNLPLAB/MouSi#2]
I try to replicate the result of ConvNext, but the accuracy is much worse than the result in Table 2. Could you please clarify which pre-trained model did you use? I use the facebook/convnext-large-224.

Besides, could you please clarify the pre-trained models of other single-experts, like SAM, DINOv2, etc. Thanks in advance :)

**Comment by cnxupupup** [comment]
Thanks for the question. The use of the visual coder model is explained in the footnotes of the paper in arxiv.

**Comment by Richar-Du** [comment]
My apologies for missing it. Thank you.
